recipes/pytorch-cpu/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pytorch-cpu
```

Comment:
Sorry I did not catch this the first time...

Shouldn't the base recipe be called pytorch, and then have an output called pytorch-cpu?

Reply:
Good point. I thought about that possibility, but in my mind: 

1. split packages have been a pain to get approved.
2. Build variants in conda-forge don't do well with the linter. https://github.com/conda-forge/conda-smithy/issues/930 https://github.com/conda-forge/conda-smithy/issues/891
2. Conda forge doesn't compile with or link to any GPU stuff. Therefore, I really don't people thinking they might be installing `pytorch-gpu` accidentally.
3. Keeping this name makes it really easy to later upgrade build variants where a gpu and cpu version might coexist in conda-forge.

We can eventually deprecate this feedstock (pytorch-cpu) if pytorch-gpu becomes a thing for conda-forge.